CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [1.0.8]
+### Bug fixes ###
+- Fix snapshot image size for pixel ratios less than 1
```

Comment:
Please take a look at the latest commits and update the bug fixes section accordingly